apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__line_precise_config_errors_bad_type.snap

```diff
@@ -10,6 +10,6 @@ configuration had errors:
     # The socket address and port to listen on
     # Defaults to 127.0.0.1:4000
     listen: true
-            ^----- true is not valid under any of the given schemas
+            ^----- true is not valid under any of the schemas listed in the 'anyOf' keyword
```

Comment:
This change in the upstream `jsonschema` dep does bring this somewhat material UX change to the _validation message_ that we show here on this property.   I'm not clear that it's actually any _better_ for the end-user, since `anyOf` is arguably something that our Router users won't understand, but I think it's also worth updating our dependencies to the latest versions, and I suppose I'm okay with the trade-off.

Though if we think another change on top of this would make this more approachable, I'm into the idea.

Reply:
Agree, let's raise a ticket for it.

Reply:
https://github.com/apollographql/router/issues/3635

Reply:
thank you!